### ISSUE TITLE
Twig extractor returns the same message over and over in file where it doen't exist

### ISSUE THREAD

**Issue by sowbiba** [PrestaShop/TranslationToolsBundle#108]
When trying to translate a module, some messages in twig templates are not visible in the BO interface

Debugging the TwigExtractor class, it seems that we have one message which come back in templates where it doesn't belong

See the video 

https://user-images.githubusercontent.com/7426640/187469128-288b0e1b-1ca6-4acc-b37e-f2415ffb7f3b.mp4

The message "It looks like we have trouble connecting to Addons. Please refresh the page or check your firewall configuration." belongs to error.html.twig but the extractor found it again in layout-ajax.html.twig and catalog-refresh.html.twig
But this string is not present in these files

On the other hand we have a lot of wordings which are not extracted

**Comment by eternoendless** [comment]
Has this been fixed by #107?

**Comment by atomiix** [comment]
I think so! ping @jolelievre @sowbiba

**Comment by jolelievre** [comment]
@atomiix @eternoendless @sowbiba it should be yes, though it's not merged in the core yet, here's the PR https://github.com/PrestaShop/PrestaShop/pull/29497 waiting for QA

@sowbiba let me know if this fixes your issue

**Comment by sowbiba** [comment]
Yes @jolelievre It fixes my issue. All twig files are correctly parsed and the catalogue is complete now

**Comment by jolelievre** [comment]
OK, then maybe we can close this issue And wait for the PR to be merged in the core (it's already associated to another Core issue)
